src/codecs/music_wavpack.c

```diff
@@ -66,7 +66,8 @@ typedef struct {
     uint32_t libversion;
     uint32_t (*WavpackGetLibraryVersion)(void);
     char *(*WavpackGetErrorMessage)(WavpackContext*);
-    WavpackContext *(*WavpackOpenFileInput)(const char *infilename, char *error, int flags, int norm_offset);
+    /* WavpackOpenFileInput is not supported on all platforms (psp and ps2 are missing ftruncate) */
+    /* WavpackContext *(*WavpackOpenFileInput)(const char *infilename, char *error, int flags, int norm_offset); */
```

Comment:
Just remove it instead of commenting it out: I can't remember why I included that function at all (probably is a remnant of some early tests of mine..)

Reply:
And remote the comment as well?

Reply:
Yep